appvm-scripts/usrbin/qubes-session

```diff
@@ -75,11 +75,5 @@ fi
 # Wait some time to pick up zombie children
 wait
 
-# Now hang myself so Xorg may live :'(
-# ____
-# |/ |
-# |  O
-# | /|\
-# | / \
-# |
-env sleep inf
+# Sleep forever so that Xorg keeps running
+exec sleep inf
```

Comment:
Please keep it separate process (no `exec`), there may be some background children that could confuse sleep when exiting.